### PR TITLE
Feature/add unit tests

### PR DESCRIPTION
**Pull request by fyoungqualabs**
## Description
Please describe the changes.
Added helper functions unit tests found on hls.js repo that were missing in the common media library.
The methods tested are:
- readId3Size
- isId3Header
- isId3Footer

## Requirements Checklist
- [ ] Unit Tests updated or fixed

### CODE REVIEW

**Review by littlespex — CHANGES_REQUESTED** [review]
Two changes:
 - The new tests are in wrong folder. Move them from `lib/test/structuredfield/` to `lib/test/id3/`.
 - Please fix the DCO sign-offs

**Review by mpollingerQualabs — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Changelog
+
```
Comment (littlespex) [inline CHANGELOG.md:2]: Please revert all whitespace changes in this file, especially the trailing whitespace in link section at the bottom

**Review by littlespex — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,177 +20,185 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.6.2] - 2024-02-21
+
+### Added
+
+- Unit tests for id3 feature helper functions [#74](https://github.com/streaming-video-technology-alliance/common-media-library/pull/74)
```
Comment (littlespex) [inline CHANGELOG.md:27]: Unfortunately changing the whitespace in the CHANGELOG will break one of our build scripts, `/scripts/createReleaseNotes.ts`. Either revert all of the white space changes, including the trailing spaces in the links section, or refactor the release notes script to account for the new formatting.

**Review by littlespex — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (fyoungqualabs) [reply CHANGELOG.md:26]: Just fixed the requested changes. Now another question came to my mind when running the script is: 
Should we append these updates to version 0.6.2 as an additional note, or would it be better to introduce a new version (0.6.3) for these changes? What do you think about that?

Comment (littlespex) [reply CHANGELOG.md:26]: Put them in the `Unreleased` section and we will move them in the next release.

Comment (fyoungqualabs) [reply CHANGELOG.md]: okay great! I just made the change.

**Review by littlespex — APPROVED** [review]

**Pull request merged**